File: Assignment2_PF/Python_Programs/P7_SubjectsMarks_Total_and_Average.py
# Output Preview Link:
# https://www.programiz.com/online-compiler/22kQ5K7yDfXog

roll_no = input("Enter Roll No: ")

marks_PF = int(input("Enter marks for Programming Fundamentals (out of 100): "))
marks_IC = int(input("Enter marks for Introduction to Computing (out of 100): "))
marks_CG = int(input("Enter marks for Computer Graphics (out of 100): "))

total = marks_PF + marks_IC + marks_CG
average = total / 3.0

print("\nRoll No:", roll_no)
print("Total Marks:", total)
print("Average Marks:", round(average, 2))

File: Assignment2_PF/C++_Programs/P7_SubjectsMarks_Total_And_Average.cpp
// Output Preview Link:
// https://onlinegdb.com/qFcG6otZ6

#include <iostream>
using namespace std;

int main() {
    int rollNo;
    float marks1, marks2, marks3, total, average;

    cout << "Enter Roll Number: ";
    cin >> rollNo;

    cout << "Enter marks for Programming Fundamentals: ";
    cin >> marks1;

    cout << "Enter marks for Introduction to Computing: ";
    cin >> marks2;

    cout << "Enter marks for Computer Graphics: ";
    cin >> marks3;

    total = marks1 + marks2 + marks3;
    average = total / 3;

    cout << "\nStudent Roll No: " << rollNo << endl;
    cout << "Total Marks: " << total << endl;
    cout << "Average Marks: " << average << endl;

    return 0;
}
